### PR TITLE
AG-11335 Assertive aria announcer

### PR DESCRIPTION
**Pull request by olegat**
https://ag-grid.atlassian.net/browse/AG-11335

### CODE REVIEW

File: packages/ag-charts-community/src/chart/chart.ts [hunk 1]
```diff
@@ -591,7 +591,7 @@ export abstract class Chart extends Observable implements AgChartInstance {
 
                 const { seriesRect } = this;
                 await Promise.all(seriesToUpdate.map((series) => series.update({ seriesRect })));
-                setAriaLabel(this.ctx.scene.canvas.container, this.getAriaLabel());
+                setAttribute(this.ctx.scene.canvas.container, 'aria-label', this.getAriaLabel());
```
Comment (alantreadway) [inline packages/ag-charts-community/src/chart/chart.ts]: Nit: I think you can just use `this.element` as the first argument here, as it's the same element?

**Review by alantreadway — COMMENTED** [review]
LGTM other than one nit

File: packages/ag-charts-community/src/chart/chart.ts (hunk 1 above)
Comment (olegat) [reply packages/ag-charts-community/src/chart/chart.ts]: Done.

**Review by alantreadway — APPROVED** [review]
LGTM

**Pull request merged**